dist/backdraft.js

```diff
@@ -894,6 +934,20 @@ _.extend(Plugin.factory, {
       </div>\
       ', null, DEFAULT_JST_DELIMS),
 
+      afterRender: function() {
+        var filterArray = JSON.parse(this.parent.table._getFilteringSettings()) || [];
+        // if there are filters in the url...
+        if (filterArray.length > 0) {
+          // find the filters that match this filter instance
+          var matches = _.where(filterArray, {type: "string", attr: this.attr});
+          // if there are url params for this filter...
```

Comment:
What does the ... mean? I think you mean is this: // if there are url params for this filter then enable those filters in the UI